backend/fms_core/template_importer/importers/transfer.py

```diff
@@ -0,0 +1,67 @@
+from fms_core.models import Protocol, Process
+from ._generic import GenericImporter
+from fms_core.template_importer.row_handlers.transfer import TransferRowHandler
+from .._utils import float_to_decimal
+
+class TransferImporter(GenericImporter):
+    SHEETS_INFO = [
+        {'name': 'SampleTransfer', 'header_row_nb': 5},
+    ]
+
+    def __init__(self):
+        super().__init__()
+        # Preload objects accessible to the whole template (not only by row)
+        self.preload_data_from_template()
+
+
+    def preload_data_from_template(self):
```

Comment:
Rename the preload_data_from_template function to whats been changed in the root template-importer branch